### ISSUE TITLE
3d_lowres performing better than 3d_cascade_fullres and 3d_fullres

### ISSUE THREAD

**Issue by annukkaa** [MIC-DKFZ/nnUNet#801]
Hello,

After running 3d_fullres on my dataset and getting low Dice scores, I decided to run 3d_lowres and 3d_cascade_fullres models to see if I could get an improvement. 

While still relatively low, I was surprised to see that the 3d_lowres scores alone were much better than either 3d_fullres or 3d_cascade_fullres.

These are the mean dice scores on the validation set of fold0 (I ran 3d_lowres for all 5 folds, but here I'm showing just the dice scores for fold 0, for a fairer comparison):

- 3d_fullres: 0.25
- 3d_lowres: 0.39
- 3d_cascade_fullres: 0.24

I'm now wondering what could be the reason for this and if you have observed this behavior before. 

I believe I'm underfitting, but increasing the number of epochs does not lead to an improvement, which I think happens because the learning rate is dependent on the number of epochs. 

Thank you,
